app/javascript/components/PartneringOrganizationProfile.js

```diff
@@ -0,0 +1,60 @@
+import React from "react"
+import PropTypes from "prop-types"
+
+/**
+*
+@prop partnering_organization -partnering organization object
+@prop all_resources
+*/
+
+class PartneringOrganizationProfile extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {
```

Comment:
So just a performance thing, we typically keep `props` in `props` (don't move them to state) unless we modify them later in the component (so here we would just keep these variables in props)